mono/utils/mono-hwcap-ppc.c

```diff
@@ -26,7 +26,21 @@
 #include <string.h>
 #include <sys/auxv.h>
 #elif defined(_AIX)
+
 #include <sys/systemcfg.h>
+
+#if !defined(POWER_4_ANDUP)
+#define POWER_4_ANDUP (POWER_4|POWER_5)
+#endif
+
+#if !defined(__power_4_andup)
+#define __power_4_andup() (_system_configuration.implementation & POWER_4_ANDUP)
+#endif
+
+#if !defined(__power_5_andup)
+#define __power_5_andup() (_system_configuration.implementation & POWER_5)
```

Comment:
This should be `POWER_5_ANDUP`, not `POWER_5`

Reply:
I'll do that later tonight - currently tweaking a specfile on i so RPMs can be churned out.

Reply:
I'll leave it to @friedkiwi since I haven't signed the CLA and the .NET Foundation CLA is not in the list of pre-approved CLAs, I'd need to get legal approval to sign it. Not worth it for a 1-line change.